### PR TITLE
bugzilla: add support for "runtime testing required" field

### PR DESCRIPTION
**Pull request by arthurzam**
Manually edited the DB dump, and updated all VCR entries by deleting them and running tests with local running instance.

Maybe `BugRuntimeTestingState` is not the best name...

Resolves: #72

### CODE REVIEW

**Comment by arthurzam** [comment]
Looks like I'm failing on the qa side, with too long lines. I tried to break it, but still fail. If you could find the best solution (I myself always hate the line too long constraint)

File: nattka/bugzilla.py [hunk 1]
```diff
@@ -73,6 +74,13 @@ def to_products_components(cls,
             assert False, f'Incorrect BugCategory: {val}'
 
 
+class BugRuntimeTestingState(enum.Enum):
+    UNSET = "---"
```
Comment (mgorny) [inline nattka/bugzilla.py:78]: I'd say start by removing this magic member and make it into `typing.Optional[] = None` instead.

Comment (arthurzam) [reply nattka/bugzilla.py]: Done

**Comment by arthurzam** [comment]
> Looks like I'm failing on the qa side, with too long lines. I tried to break it, but still fail. If you could find the best solution (I myself always hate the line too long constraint)

Found the correct solution. I needed to surround it with `(` and `)`

**Review by mgorny — APPROVED** [review]
LGTM

**Pull request closed without merging**